parachain/pallets/eth-app/src/lib.rs

```diff
@@ -108,10 +108,11 @@ impl<T: Trait> Module<T> {
 }
 
 impl<T: Trait> Application for Module<T> {
-	fn handle(payload: &[u8]) -> DispatchResult {
+	fn handle(payload: &[u8], _verification_output: &VerificationOutput) -> DispatchResult {
 		let payload_decoded = InPayload::decode(payload)
 			.map_err(|_| Error::<T>::InvalidPayload)?;
-
+		// TODO: check that payload event exists in VerificationOutput.Receipt
```

Comment:
can we add github issue with security flag for this, rather than TODO, as its pretty important not to miss.

Reply:
This is taken care of in https://github.com/Snowfork/polkadot-ethereum/pull/184